tests/test_processing.py

```diff
@@ -49,13 +49,18 @@
     }
 }
 NOW = datetime.datetime.now()
+INVALID_TITLE_ERROR = processing.ValidationError(name="InvalidTitle", message="title is too long")
 JOB_METADATA: processing.JobMetadata = {
     "processID": PROCESS_ID,
     "jobID": JOB_ID,
     "accountID": ACCOUNT_ID,
     "workspaceID": constants.WORKSPACE_ID,
     "definition": DEFINITION,
-    "results": {"collection": f"{COLLECTION_URL}"},
+    "results": {
+        "collection": f"{COLLECTION_URL}",
+        "errors": [dataclasses.asdict(INVALID_TITLE_ERROR)],
+    },
+    "creditConsumption": {"credits": CREDITS},
```

Comment:
`hold_id` and `consumption_id` are not included. It is OK, but just a remark considering that in the JobTemplate we created a class for JobCost. I was wondering why we chose a different approach here.